pandas/tests/arrays/boolean/test_logical.py

```diff
@@ -0,0 +1,230 @@
+import operator
+
+import numpy as np
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+from pandas.arrays import BooleanArray
+from pandas.tests.extension.base import BaseOpsUtil
+
+
+class TestLogicalOps(BaseOpsUtil):
```

Comment:
Side comment: do we need these classes in general? Seems like the location / name of the test should give enough context

Reply:
In general, if there is only a single class in a file, certainly not. But in this case, it might use things from the BaseOpsUtil class

Reply:
agreed with @jorisvandenbossche.  Largely its a matter of personal preference; I tend to retain the class structure in cases where it helps further organize tests